content/departments/cloud/cloud-cost.md

```diff
@@ -0,0 +1,35 @@
+# Cloud Cost Policy
+
+When designing features that will exist on Sourcegraph-managed instances we should consider if there will be an increase in cost. This document is primarily focused on infrastructure costs but you should also provide justification that costs will not increase with regards to:
+
+- Setup time (will this feature make Sourcegraph harder to setup?)
+- Support costs (has the customer support team been engaged and have they signed off on supporting this new feature?)
+- Complexity (is this feature still being tested and requires additional steps)
+
+If any of the above is true, this shouldn’t prevent you from launching on Cloud as long as your team will work closely with ours and this feature is clearly flagged as experimental.
+
+# Requirements
+
+Utilize **feature flags** (ie through site-config or experimental flags) such that a feature can be enabled or disabled as needed. This also allows for the Cloud team to roll out the changes using “canary” methodologies to limit the blast radius for managed instances.
+
+Monitor and test features on S2. S2 has continous deployment so changes that are pushed to the `main` branch will be deployed to S2. Its not always possible to determine how a change may affect the overall cost. It is the feature team’s responsibility to check the billing page to ensure that their feature is not causing any large spikes in cost. The billing view is saved [here](https://console.cloud.google.com/billing/017005-C370B2-0E3030/reports;savedView=8549f3e5-8bfa-4a88-9b41-42a72eeb93f5?organizationId=244397465763&project=sourcegraph-dev).
```

Comment:
I would explicitly mention to check Grafana i.e for errors or [excessive usage of resources](https://sourcegraph.sourcegraph.com/-/debug/grafana/d/containers/global-containers-resource-usage?orgId=1) and compare to last 2 weeks.